src/joyride/sci.cljs

```diff
@@ -48,6 +48,10 @@
        :path-to-load path-to-load
        :source (str (fs/readFileSync path-to-load))})))
 
+(defn source-script-by-ns [namespace]
+  (or (source-script-by-ns-with-ext namespace ".cljc")
+      (source-script-by-ns-with-ext namespace ".cljs")))
```

Comment:
I think these should be reserved, in CLJS `.cljs` is preferred over `.cljc`.
(In Clojure `.clj` is preferred over `.cljc`)